pbs_light.go

```diff
@@ -113,6 +115,7 @@ var (
 var exchanges map[string]adapters.Adapter
 var dataCache cache.Cache
 var reqSchema *gojsonschema.Schema
+var CurrencyRates atomic.Value
```

Comment:
If possible, we would like to reduce global variables, not add more. :)

Reply:
Also why are we using sync.atomic when currency rates are only loaded on startup before we are ready to serve any requests? I don't see any possibility for race condition updates.

Reply:
feedback from @dbemiller included making this an atomic value, to be able to use Store() in a threadsafe way.